### PR TITLE
Move smallft to 3rdparty/smallft-src and add to 3rdPartyLicenses.

### PR DESCRIPTION
**Pull request by mkrautz**
The smallft code is not under Mumble's license, so it should
live under 3rdparty, and have a proper license notice in our
binaries.

### CODE REVIEW

File: src/mumble/mumble.pro [hunk 1]
```diff
@@ -210,12 +210,12 @@ SOURCES *= BanEditor.cpp \
     VoiceRecorderDialog.cpp \
     WebFetch.cpp \
     MumbleApplication.cpp \
-    smallft.cpp \
+    ../../3rdparty/smallft-src/smallft.cpp \
     ThemeInfo.cpp \
     Themes.cpp \
     OverlayPositionableItem.cpp
 
-DIST		*= ../../icons/mumble.ico licenses.h smallft.h ../../icons/mumble.xpm murmur_pch.h mumble.plist
```
Comment (hacst) [inline src/mumble/mumble.pro]: Does DIST do anything for us these days? Could/Should we remove it completely?

**Comment by hacst** [comment]
LGTM

File: src/mumble/mumble.pro (hunk 1 above)
Comment (mkrautz) [inline src/mumble/mumble.pro]: Nope.

It was used to add extra distfiles for "make dist".
But we haven't used make dist in a long time.
We use scripts/release.pl instead.
And that no longer reads DIST from pro files. It works from the assumption that everything that isn't explicitly blacklisted is in.

So yes, should be able to remove 'em. And I think we should. :-)

**Comment by mkrautz** [comment]
Landed as 5c17181b02e86d6bb61d17f3d4341bc2904a766c.

**Pull request closed without merging**